cmd/vsphere-xcopy-volume-populator/e2e-tests/Dockerfile

```
@@ -0,0 +1,84 @@
+FROM golang:1.24 AS builder
+
+WORKDIR /forklift
+
+# Copy go module files from the context (project root)
+COPY go.mod go.sum ./
+RUN go mod vendor
+
+# Copy go module files for the populator
+COPY cmd/vsphere-xcopy-volume-populator/go.mod cmd/vsphere-xcopy-volume-populator/go.sum ./cmd/vsphere-xcopy-volume-populator/
+RUN cd cmd/vsphere-xcopy-volume-populator && go mod vendor
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)